.github/workflows/code-validation.yaml

```diff
@@ -0,0 +1,20 @@
+name: Code Validation
+
+on: [push]
```

Comment:
Is this set for a specific branch ? Cause for now this will trigger one every branch and every push? Do you want to narrow it down or is it intended ?

Reply:
Could do tbf, lemme change it so it's just for PRs